CMakeLists.txt

```diff
@@ -292,7 +292,7 @@ if (STATIC)
 	if (NOT DISABLE_TTF)
 		# FreeType is required by SDL2_ttf, but not wired up properly in package
 		PKG_CHECK_MODULES(FREETYPE REQUIRED freetype2)
-	endif (DISABLE_TTF)
+	endif (NOT DISABLE_TTF)
```

Comment:
@marijnvdwerf shoo, you don't even use cmake :P.

Seriously though, no real reason, it already had the condition in it and it's *slightly* more informative.